dockerfiles/prolog/latest/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:bullseye-slim
```

Comment:
We prefer our docker images to base off of Ubuntu. Should be able to easily swap it for `ubuntu:22.04`.

Reply:
oh I just copied what the verilog file does but yes I'll swap it